client/webserver/site/src/js/markets.js

```diff
@@ -400,11 +403,30 @@ export default class MarketsPage extends BasePage {
   handleBookRoute (data) {
     const market = this.market
     const page = this.page
-    const [b, q] = [market.base, market.quote]
-    if (data.base !== b.id || data.quote !== q.id) return
+    const url = market.dex.url
+    let [b, q] = [market.base, market.quote]
```

Comment:
Just use the `user.exchanges` asset here to start. We don't actually care here whether the asset is supported.